tests/baselines/reference/autoAccessorNoUseDefineForClassFields.errors.txt

```diff
@@ -0,0 +1,49 @@
+file3-2.ts(1,7): error TS2300: Duplicate identifier 'C3'.
+file3.ts(1,7): error TS2300: Duplicate identifier 'C3'.

```

Comment:
These seem unintentional; do the tests need to use different names if they are previously using global scripts?

Reply:
Yes. It was probably unintentional. The original test had a file with the same name, and that was causing some issues. I renamed the class